client/dist/bundle.js

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
avoid committing bundle.js to GitHub, add it to gitignore

Reply:
good point! I think you are right about the package JSON also